config/locales/client.en.yml

```diff
@@ -542,3 +545,4 @@ en:
         no_results: "No similar topics."
         loading: "Looking for similar topics..."
         show: "show"
+        
```

Comment:
@keegangeorge the blank line should be exactly that, just a NL char, that's it, no padding, (I suspect that's why the GitHub interface is flagging it again).